workers/run_tests.sh

```diff
@@ -11,11 +11,19 @@ cd $script_directory
 # move up a level
 cd ..
 
+# Set up the test data volume directory if it does not already exist
+volume_directory="$script_directory/test_volume"
+if [ ! -d "$volume_directory" ]; then
+    mkdir $volume_directory
```

Comment:
I agree that this is a very rare case. So it is okay if you don't want to handle it.